pkg/apis/monitoring/v1/types.go

```diff
@@ -545,6 +548,57 @@ type ArbitraryFSAccessThroughSMsConfig struct {
 	Deny bool `json:"deny,omitempty"`
 }
 
+type PrometheusTracingConfig struct {
+	// Client used to export the traces. Options are 'http' or 'grpc'.
+	//+kubebuilder:validation:Enum=http;grpc
+	// +optional
+	ClientType PrometheusTracingClientType `json:"clientType"`
```

Comment:
Unless the same type is used elsewhere, I don't think we need a `PrometheusTracingClientType` type.

```suggestion
	ClientType string `json:"clientType"`
```

Reply:
Yep sounds fair 👍 

This is not required to add validation at the CRD level, right?